package.json

```diff
@@ -33,31 +12,55 @@
     "markdown",
     "prettier"
   ],
-  "scripts": {
-    "build": "rimraf *.d.ts && tsc -b && type-coverage",
-    "lint": "run-p lint:*",
-    "lint:es": "eslint . --cache -f friendly",
-    "lint:tsc": "tsc --emitDeclarationOnly false --noEmit",
-    "prelint": "yarn build",
-    "prepare": "simple-git-hooks && yarn-deduplicate --strategy fewer || exit 0",
-    "typecov": "type-coverage"
+  "repository": "remarkjs/remark-preset-prettier",
+  "bugs": "https://github.com/remarkjs/remark-preset-prettier/issues",
+  "funding": {
+    "type": "opencollective",
+    "url": "https://opencollective.com/unified"
+  },
+  "author": "JounQin <admin@1stg.me>",
+  "contributors": [
+    "JounQin <admin@1stg.me>",
+    "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
+  ],
+  "sideEffects": false,
+  "type": "module",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.d.ts",
+    "index.js"
+  ],
+  "exports": {
+    "types": "./index.d.ts",
+    "default": "./index.js"
   },
   "peerDependencies": {
     "prettier": ">=1.0.0"
   },
   "devDependencies": {
-    "@1stg/lib-config": "^10.2.1",
-    "@types/node": "^18.7.13",
-    "@types/prettier": "^2.7.0",
-    "type-coverage": "^2.22.0",
-    "typescript": "^4.8.2",
-    "yarn-deduplicate": "^5.0.2"
+    "prettier": "^2.0.0",
+    "type-coverage": "^2.0.0",
+    "typescript": "^4.0.0",
+    "unified": "^10.0.0",
+    "xo": "^0.52.0"
```

Comment:
I don't like xo, really, and it does not check `.json`, `.md` and `yml`, etc. files at all.

Reply:
It enforces a strict, useful, code-style. It prevents:
a) bugs,
b) having to discuss code style

Reply:
My current `@1stg/eslint-config` already does this, and check `.json`, `.md`, `.yml`, etc. at the same time.

Reply:
Your personal code style is not consistent with the rest of the projects maintained by us.

Reply:
> Your personal code style is not consistent with the rest of the projects maintained by us.

`xo` is chosen by you personally as I mentioned at #99 

And also

> But I'm also totally fine with a new `eslint-config-unifiedjs`.

> ESLint/stylelint/prettier are the standard way and very flexible while xo is definitely not.

Reply:
> ESLint/stylelint/prettier are the standard way and very flexible while xo is definitely not.

ESLint is used in `xo`. `xo` provides more things than ESLint does.
Most notably, it provides *not having to discuss this* and getting on with fixing bugs.
The goal of `xo` is not to be as flexible as `eslint`.
`xo` is more standard than your personal code style.

Murderlon addressed this in https://github.com/remarkjs/remark-preset-prettier/pull/99#issuecomment-1216387617.

> But I'm also totally fine with a new eslint-config-unifiedjs.

Feel free to discuss this somewhere where everyone can participate about how we should lint 300 projects.
I am personally probably against it. `xo` already has good rules. `xo` takes away having to discuss these things.

> xo is chosen by you personally as I mentioned at https://github.com/remarkjs/remark-preset-prettier/pull/99

Which sentence are you talking about?

It isn’t about personal preference of code style. That’s the whole point. It is about consistency and not having to discuss it

Reply:
The problem is `xo` doesn't handle a lot of other files like `.json`, `.yml`, etc. by https://github.com/ota-meshi/eslint-plugin-json-schema-validator for example. I'd like to have `eslint-config-unifiedjs` including `eslint-config-xo` inside with addition `overrides` for other files.

`xo` is doing well for `.js`/`.ts`, but we have other source files.